code/dynamic_programming/factorial/factorial.cpp

```diff
@@ -0,0 +1,31 @@
+/* 
+ * C++ Program to Find Factorial of a Number using Dynamic Programming 
+ */
+#include <iostream>
+#define ll long long
```

Comment:
This preprocessor macro is not used in the preprocessing phase. Also, the short name `ll` does not improve readability for me. If you like to rename types, checkout C++11 type aliases.